src/blog/2023/09/dashboard-notebook-layout.md

```diff
@@ -9,6 +9,7 @@ tags:
     - posts
     - releases
     - community
+    - changelog
```

Comment:
I've added this is a demo of cross-posting between main blog and changelog. Don't necessarily propose doing this - but wanted to show it was possible whilst discussing this PR.